pandas/tests/groupby/transform/test_transform.py

```diff
@@ -188,6 +190,8 @@ def test_transform_axis_1(request, transformation_func, using_array_manager):
     tm.assert_equal(result, expected)
 
 
+# TODO(ArrayManager) groupby().transform returns DataFrame backed by BlockManager
+@td.skip_array_manager_not_yet_implemented
```

Comment:
This might be solved with latest master?

Reply:
yep, will push shortly